### DISCUSSION TITLE
temporary override for switching on the heaters

### DISCUSSION THREAD

**Discussion by andreapede** [jmcollin78/versatile_thermostat#643] [Q&A] [answered]
I'm using this fantastic component with the suggested scheduler.
 I have set up different presets depending on the day. I would like to know the best approach to temporarily force a heating activation (for example, by selecting 'booster') that has a precise end time and does not interfere with the normal scheduled behavior.
How could do it?

**Comment by jmcollin78** [comment, marked as answer]
Hello,

Interesting use case and here is how you can do in an automation:
1. when you want to start a preset for a period of time, the first thing to do is to deactivate the Scheduler. There is a switch associated with the Scheduler, so you can turn it off,
2. then switch the preset in VTherm,
3. wait let says one hour,
4. re-enable the Scheduler. It should then send the current programmed preset to the VTherm.


Tell me if this works as is.

Note: this could be an interesing blueprint for the community

**Reply by andreapede** [reply]
Many thanks.
The switching on/off of the scheduler was the missing puzzle piece.
I implemented everything like suggested
Basically I added a button for triggering the +1 action, switch off the scheduler and start the timer and pushing to "boost" the thermostat
at the end I simply switched on again the scheduler and everything returned to normal state.
I also created a timer card that shows the remaining override time and in case of pressing it will force the override to finish anticipating the timer.
thank you really much 
Andrea
